targets/TARGET_NUVOTON/TARGET_M480/mbed_overrides.c

```diff
@@ -86,52 +72,29 @@ void mbed_sdk_init(void)
     /* Lock protected registers */
     SYS_LockReg();
 
-    /* Get around h/w issue with reset from power-down mode
+    /* Get around h/w issue with reset from deep power-down mode
      *
      * When UART interrupt enabled and WDT reset from power-down mode, in the next
      * cycle, UART interrupt keeps breaking in and cannot block unless via NVIC. To
-     * get around it, we make up a signal of WDT wake-up from power-down mode in the
+     * get around it, we make up a signal of wake-up from deep power-down mode in the
      * start of boot process on detecting WDT reset.
      */
     if (SYS_IS_WDT_RST()) {
-        /* Re-unlock to highlight WDT clock setting is protected */
+        /* Re-unlock protected clock setting */
         SYS_UnlockReg();
 
-        /* Enable IP module clock */
-        CLK_EnableModuleClock(WDT_MODULE);
+        /* Set up DPD power down mode */
+		CLK->PMUSTS |= CLK_PMUSTS_CLRWK_Msk;
+		CLK->PMUSTS |=	CLK_PMUSTS_TMRWK_Msk;
+		CLK_SetPowerDownMode(CLK_PMUCTL_PDMSEL_DPD);
 
-        /* Select IP clock source */
-        CLK_SetModuleClock(WDT_MODULE, CLK_CLKSEL1_WDTSEL_LIRC, 0);
-
-        /* The name of symbol WDT_IRQHandler() is mangled in C++ and cannot
-         * override that in startup file in C. Note the NVIC_SetVector call
-         * cannot be left out when WDT_IRQHandler() is redefined in C++ file.
-         *
-         * NVIC_SetVector(WDT_IRQn, (uint32_t) WDT_IRQHandler);
-         */
-        NVIC_EnableIRQ(WDT_IRQn);
-
-        /* Configure/Enable WDT */
-        WDT->CTL = WDT_TIMEOUT_2POW4 |      // Timeout interval of 2^4 LIRC clocks
-                WDT_CTL_WDTEN_Msk |         // Enable watchdog timer
-                WDT_CTL_INTEN_Msk |         // Enable interrupt
-                WDT_CTL_WKF_Msk |           // Clear wake-up flag
-                WDT_CTL_WKEN_Msk |          // Enable wake-up on timeout
-                WDT_CTL_IF_Msk |            // Clear interrupt flag
-                WDT_CTL_RSTF_Msk |          // Clear reset flag
-                !WDT_CTL_RSTEN_Msk |        // Disable reset
-                WDT_CTL_RSTCNT_Msk;         // Reset up counter
+        /* Set up PMU wakeup timer, wakeup interval must be WKTMRIS_256 25.6 ms at least */
+        CLK_SET_WKTMR_INTERVAL(CLK_PMUCTL_WKTMRIS_256);
```

Comment:
are these lines (91-93) misaligned (compare to 89 and above) ?

Reply:
@0xc0170 , thanks of your reminding, fixed in  commit 7db8549.